stock-tracker/src/store/subscriptions.js

```diff
@@ -0,0 +1,60 @@
+import {
+  setChartDataAction,
+  setChartErrorAction
+} from "../features/chart/redux/actions";
+import {
+  setCompanyOverviewAction,
+  setErrorOverviewAction
+} from "../features/overview/redux/actions";
+import {
+  setLatestNewsAction,
+  setErrorNewsAction
+} from "../features/latestNews/redux/actions";
+import {
+  setResponseAction,
+  setErrorKeyStatsAction
+} from "../features/keyStats/redux/actions";
+import {
+  addTopPeersAction,
+  setErrorPeersAction
+} from "../features/topPeers/redux/actions";
+import {
+  STOCK_DATA,
+  COMPANY_OVERVIEW,
+  LATEST_NEWS,
+  SUGGESTIONS,
+  CHART_DATA,
+  TOP_PEERS,
+  COMPANY_OVERVIEW_ERROR,
+  LATEST_NEWS_ERROR,
+  CHART_DATA_ERROR,
+  TOP_PEERS_ERROR,
+  STOCK_ERROR
+} from "../socket/eventTypes";
+import { setSuggestionsAction } from "../features/search/redux/actions";
+import { socketService } from "../socket";
+
+export const createSocketSubscriptions = dispatch => {
+  socketService.create();
+  const unsubscribeFns = [
```

Comment:
This array of `[action type, action]` tuples could be extracted out of this function into a constant above.
You could also change it to a map, if you wanted to make sure each action type is unique and handled only once, but that's preference.